src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
+@import './utils/animation';
+
 body {
   margin: 0;
+  padding: 0;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin #000 dotted;
+  border-radius: 50%;
+
+  &__center {
+    height: 5vmin;
+    width: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    position: absolute;
+  }
+
+  &__second-hand {
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    position: absolute;
+
+    transform-origin: bottom;
+    animation: stopwatch-hand-animation 60s linear infinite;
```

Comment:
The `animation` name 'stopwatch-hand-animation' does not follow the BEM naming convention. Consider using a name that reflects the BEM structure of the element it's applied to, like 'stopwatch__second-hand--animation'.